src/pages/Notification/Notifications.tsx

```diff
@@ -1,65 +1,56 @@
 /* eslint-disable jsx-a11y/no-static-element-interactions */
 /* eslint-disable jsx-a11y/click-events-have-key-events */
 import { Button, Typography } from "antd";
-import React, { useEffect, useState } from "react";
+import React from "react";
+import NotificationBox, { Container } from "pages/Notification/style";
 import {
-  useSendNotificationMutation,
   useGetNotificationsQuery,
-  useReadNotificationMutation,
-  getSocket,
+  useReadNotificationsMutation,
 } from "services/notifications/setNotificationsAPI";
-import { Notifications as NotificationsType } from "services/notifications/chatTypes";
-
-import S from "pages/Notification/style";
+import { formatToStandardDate } from "utils/functions/formatDateFunctions";
+import { useNavigate } from "react-router-dom";
+import { CHAT_ROUTE } from "utils/consts/routeConsts";
+import { useTranslation } from "react-i18next";
 
 export default function Notifications() {
-  const [sendNotification] = useSendNotificationMutation();
-  const [readNotification] = useReadNotificationMutation();
-  const { data, isLoading, isSuccess } = useGetNotificationsQuery();
-  const [notifications, setNotifications] = useState<NotificationsType[]>([]);
-
-  useEffect(() => {
-    if (isSuccess && !isLoading && data) {
-      setNotifications(data);
-    }
-  }, [isLoading, isSuccess, data]);
+  const { t } = useTranslation();
 
-  const socket = getSocket();
+  const { data } = useGetNotificationsQuery();
 
-  function receiveNotification(notification: NotificationsType) {
-    const newNotification = { ...notification };
-    setNotifications([...notifications, newNotification]);
-  }
+  const [markNotificationsAsRead] = useReadNotificationsMutation();
 
-  socket.on("first-message", (notification) => {
-    receiveNotification(notification);
-  });
+  const navigate = useNavigate();
 
-  const handleReadNotification = (id: number) => {
-    readNotification(id);
+  const onClickHandler = (roomId: number, notificationId: number) => {
+    markNotificationsAsRead([notificationId]);
+    navigate(`${CHAT_ROUTE}/${roomId}`);
   };
 
-  const sendTestNotification = () => {
-    const newNotification = {
-      fromUserId: 1,
-      toUserId: 1,
-      type: "TEST_NOTIFICATION",
-    };
-    sendNotification(newNotification);
-  };
   return (
-    <>
-      <h1>Notifications</h1>
-      <Button onClick={sendTestNotification}>Create Notification</Button>
-      {notifications.map((item) => (
-        <S.NotificationBox
-          key={item.id}
-          onClick={() => handleReadNotification(item.id || 0)}
-        >
-          <Typography>Notification type: {item.type}</Typography>
-          <Typography>From: {item.fromUser?.firstName}</Typography>
-        </S.NotificationBox>
-      ))}
-    </>
+    <Container>
+      <h1>{t("Notifications.title")}</h1>
+      {data?.notifications?.length ? (
```

Comment:
nonono better 
```
!data?.notifications return  <div>{t("Notifications.noNotifications")}</div>
  return (
    <Container>
      <h1>{t("Notifications.title")}</h1>
....
```
if make more simple example instead
```
if(isEqual){
 calculations
calculations
calculations
calculations
calculations
calculations
calculation1
} else return 1
```

better make
```
if (!isEqual) return 1
 calculations
calculations
calculations
calculations
calculations
calculations
calculation1
```

its match more readable and understandable

Reply:
done